<launch>
  <node name="aruco_track" pkg="aruco_track" type="aruco_track" args="_publish_debug_image:=true">
    <remap from="~source" to="/file_stream/camera/image" />
    <remap from="~camera_info" to="/file_stream/camera_info" />
  </node>
</launch>
